server/constants/paymentMethod.ts

```diff
@@ -0,0 +1,2 @@
+export const services = ['paypal', 'stripe', 'opencollective', 'prepaid'];
+export const types = ['creditcard', 'prepaid', 'payment', 'collective', 'adaptive', 'virtualcard'];
```

Comment:
**[:sparkles: **Enhancement**]** I'd prefer if we can set explicit names here, like `paymentMethodServices`. Because these variables are non-mutable (*real* constants) I'd event argue they should be capitalized: `PAYMENT_METHOD_SERVICES` / `PAYMENT_METHOD_TYPES`.

For example for the collectives constants we also export a `type` variable in `server/constants/collectives.ts` and I find it very unpractical for two reasons:
1. If you're deep in a file, it may be unclear what this `type` variable is and where it's coming from
2. It prevents the variable names from being used in the whole file, and these are very common variable names

Sure you can solve that by aliasing your imports, but I'd rather have explicit names by default to simplify their usage.

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>

Reply:
For the same problem, I proposed https://github.com/opencollective/opencollective-api/pull/3623#discussion_r401182819 but your proposal is better.

Reply:
Fixed in 0ddfca9.